GooglePay/src/main/java/com/braintreepayments/api/googlepay/GooglePayInternalClient.kt

```diff
@@ -0,0 +1,48 @@
+package com.braintreepayments.api.googlepay
+
+import android.content.Context
+import com.braintreepayments.api.core.Configuration
+import com.braintreepayments.api.googlepay.GooglePayReadinessResult.NotReadyToPay
+import com.google.android.gms.common.api.ApiException
+import com.google.android.gms.tasks.Task
+import com.google.android.gms.wallet.IsReadyToPayRequest
+import com.google.android.gms.wallet.Wallet
+import com.google.android.gms.wallet.Wallet.WalletOptions
+import com.google.android.gms.wallet.WalletConstants
+
+internal class GooglePayInternalClient {
+    fun isReadyToPay(
+        context: Context?,
+        configuration: Configuration,
+        isReadyToPayRequest: IsReadyToPayRequest?,
+        callback: GooglePayIsReadyToPayCallback
+    ) {
+        val paymentsClient = Wallet.getPaymentsClient(
+            context!!,
+            WalletOptions.Builder()
+                .setEnvironment(getGooglePayEnvironment(configuration))
+                .build()
+        )
+        paymentsClient.isReadyToPay(isReadyToPayRequest!!)
```

Comment:
Same here, can we handle when `isReadyToPayRequest` is null?

Reply:
It looks like we can simply make the function parameters non-null, I will push another update shortly.